packages/docs-app/src/styles/_banner.scss

```diff
@@ -0,0 +1,59 @@
+// Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the terms of the LICENSE file distributed with this project.
+
+@import "~@blueprintjs/core/src/common/variables";
+
+$banner-height: $pt-grid-size * 4;
+
+// banner along top of screen linking to v2 docs
+.docs-banner {
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+
+  z-index: $pt-z-index-overlay - 1;
+  box-shadow: inset 0 -1px 0 $pt-divider-black;
+  background: $green3;
+  height: $banner-height;
+  text-align: center;
+  line-height: $banner-height;
+  color: $white;
+
+  &:hover {
+    background-color: $green2;
+    text-decoration: none;
+    color: $white;
```

Comment:
Do we need to redeclare `color: $white` on hover?

Reply:
not sure, I just pasted this from the internal site